cmd/juju/charmhub/convert.go

```diff
@@ -85,36 +88,26 @@ func convertBundle(charms []transport.Charm) *Bundle {
 	return bundle
 }
 
-func convertCharm(info transport.InfoResponse) (*Charm, []string, error) {
-	charmHubCharm := Charm{
+func convertCharm(info transport.InfoResponse) (ch *Charm, isKubernetes bool) {
+	ch = &Charm{
 		UsedBy: info.Entity.UsedBy,
 	}
-	var series []string
-	var err error
 	if meta := unmarshalCharmMetadata(info.DefaultRelease.Revision.MetadataYAML); meta != nil {
-		charmHubCharm.Subordinate = meta.Subordinate
-		charmHubCharm.Relations = transformRelations(meta.Requires, meta.Provides)
+		ch.Subordinate = meta.Subordinate
+		ch.Relations = transformRelations(meta.Requires, meta.Provides)
 
 		// TODO: hml 2021-04-15
 		// Implementation of Manifest in charmhub InfoResponse is
```

Comment:
Can this TODO be removed? I think it can as we're using bases from the response data rather than the manifest.